packages/headless/src/controllers/index.ts

```diff
@@ -152,6 +154,8 @@ export {
   buildSearchBox,
 } from './search-box/headless-search-box';
 
+export {CaseAssist, buildCaseAssist} from './case-assist/headless-case-assist';
```

Comment:
One thing we will need to explore is creating a dedicated export namespace for CaseAssist to avoid adding to the confusion we've already seen between SearchApp and Recommendation.

If the exports are at the same level, then people will mix-and-match incompatible controllers.

Reply:
I've got an incoming PR with an additional feature set for `user profiles` which i guess reinforces the need for this separation.

Reply:
@samisayegh I was not aware of confusion between SearchApp and Recommendation, but I do understand why it can be confusing.

I can have a quick look at providing a dedicated export for Case Assist, but I don't actually know how to implement that. So if you have some pointers you can give me, it would be much appreciated.

Reply:
Alright, we can look at how to best expose the case assist functionality as a final step. For now, let's not export the new controllers. I want to work together on the structure of  functionality. I will message you once I've gone through the API. Thanks for sharing the link,

Reply:
According to [this post](https://stackoverflow.com/questions/53680866/how-to-export-a-submodule-in-a-typescript-library), exporting a sub-module seems to be mostly a matter of having proper `index.ts` files in place to export the right symbols at the right level.